Penumbra/Import/Models/Export/MeshExporter.cs

```diff
@@ -385,6 +398,8 @@ private Type GetMaterialType(IReadOnlyDictionary<MdlFile.VertexUsage, MdlFile.Ve
 
         return materialUsages switch
         {
+            (3, true) => typeof(VertexTexture3ColorFfxiv),
+            (3, false) => typeof(VertexTexture3),

```

Comment:
L410 currently throws `"Unreachable."` as the prior impl quite literally could only ever return `0..=2` for `uvCount`. while there's absolutely no expectation that it actually will, a theoretical model with two `Half4`/`Single4` UV could hit `uvCount == 4` (or similar with any usage index > 1), so updating this to an an error via the notifier is probably pertinent.